projects/batfish-common-protocol/src/main/java/org/batfish/common/Container.java

```diff
@@ -3,66 +3,64 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.MoreObjects;
-import java.nio.file.Path;
 import java.util.Objects;
-import java.util.SortedSet;
 
+/**
+ * The {@link Container Container} is an object representation of the container for BatFish service.
+ *
+ * <p>Each {@link Container Container} contains a name and a summary of the container {@link
+ * #_name}.
+ */
 public final class Container {
   private static final String PROP_NAME = "name";
-  private static final String PROP_TESTRIGS = "testrigs";
-  private static final String PROP_ANALYSIS = "analysis";
+  private static final String PROP_CREATED_AT = "createdAt";
+  private static final String PROP_TESTRIGS_COUNT = "testrigsCount";
+  private static final String PROP_ANALYSES_COUNT = "analysesCount";
 
   private String _name;
-  private SortedSet<String> _testrigs;
-  private Path _analysis;
+  private String _createdAt;
```

Comment:
createdAt should be a date

Reply:
Jakson serializes Date as a long number by default, so I store createdAt as a string.